source/multiple-mongosyncs.txt

```diff
@@ -18,8 +18,6 @@ There are two ways to synchronize :ref:`sharded clusters
 loaded clusters, use multiple ``monogosync`` instances, one
 ``monogosync`` for each shard in the cluster.
 
-.. include:: /includes/sharding-warning
```

Comment:
[question]

Since this is an include, are there any other places this warning appears that we should remove?

Reply:
Interestingly, a quick search of the repo doesn't show this used in any other place.